### DISCUSSION TITLE
Merge cells by numbers

### DISCUSSION THREAD

**Discussion by egrittini-solinfo** [PHPOffice/PhpSpreadsheet#4245] [General]
Hi, is it possibile to merge cells by using something like that?

$spreadsheet->getActiveSheet()->mergeCells([3,1].':'.[6,1]);

**Comment by noombkk** [comment]
You can use 
$spreadsheet->getActiveSheet()->mergeCells(chr(68) . '1:' . chr(71) . 1);
chr(65) = A, chr(66) = B . . . .
Wish you success

**Comment by oleibman** [comment]
In general, you can always convert `[$column, $row]` to a form usable by `mergeCells` using:
```php
\PhpOffice\PhpSpreadsheet\Cell\CellAddress::fromColumnRowArray([$column, $row])
``` 
However, even more simply in your case, `mergeCells` will let you specify:
```php
$spreadsheet->getActiveSheet()->mergeCells([3,1,6,1]);
```

**Comment by oleibman** [comment]
Closing, question is answered.
